SimTracker/TrackerMaterialAnalysis/test/listIds.py

```diff
@@ -1,22 +0,0 @@
-#! /usr/bin/env cmsRun
-
-import FWCore.ParameterSet.Config as cms
-
-process = cms.Process("MaterialAnalyser")
-process.load("Configuration.StandardSequences.Geometry_cff")
-process.load("Configuration.StandardSequences.MagneticField_cff")
-
-process.MessageLogger = cms.Service("MessageLogger",
-    destinations = cms.untracked.vstring('cout')
-)
-
-process.source = cms.Source("EmptySource")
-process.maxEvents = cms.untracked.PSet(
-    input = cms.untracked.int32(1)
-)
-
-process.listIds = cms.EDFilter("ListIds"
```

Comment:
a comma was missing here, as you pointed out.

Reply:
Also `ListIds` is an `EDAnalyzer` not an `EDFilter`. 
Can you try the suggestion above and restore this file?